modules/backup-etcd.adoc

```diff
@@ -21,6 +21,15 @@ You should only save a backup from a single master host. You do not need a backu
 +
 ----
 $ sudo /usr/local/bin/etcd-snapshot-backup.sh ./assets/backup
```

Comment:
@retroflexer We say in step 1 to access the host as the root user, then proceed to use `sudo` in this command. Should I just remote "as the root user" from step 1 and keep `sudo` in this command? We do use `sudo` for most of the restore procedures so it is probably best to keep that if that works.

Reply:
Yes. We don't want them to access the system as root. That's why we include sudo in the commands assuming they are running as non-root user.